sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -110,6 +114,18 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
     mapController.initializePlugins(resolvedMapInitOptions, this)
   }
 
+  // need activity instance for NDK frame pacing, https://stackoverflow.com/a/32973351
+  private fun getActivity(): Activity? {
```

Comment:
if you need activity, maybe set result type as `Activity` without `?`
and it can't be found, throw exception. because you already cast `null` to activity!!